first_program/first_program/main.cpp

```diff
@@ -0,0 +1,60 @@
+#include<iostream>
+
+using namespace std;
+
+int main()
+{
+     cout << "We started arithmetic operation!" << endl;
+    char operation;
+   double firstVariable;
+   double secondVariable;
+   double result;
+    
+    cout << "Please enter the first variable: " << endl;
+    cin >> firstVariable;
+    
+    cout << "Please enter the second variable: " << endl;
+    cin >> secondVariable;
+    
+    cout << "Please enter the operation: " << endl;
+    cin >> operation;
+    
+   
+    cout << "The first variable value: " << firstVariable << endl;
+    cout << "The second variable value: " << secondVariable << endl;
+    cout << "The operation: " << operation << endl;
+    
+    switch(operation){
+        case '+':
+             result=firstVariable+secondVariable;
```

Comment:
Fix the formatting by removing the extra space.

Reply:
I have removed every extra space